parse_pom_sequence.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+
+#We want any failures to be fatal
+# Depends: maven
+
+set -e
+
+if ! `which mvn 1>/dev/null 2>&1` ; then
```

Comment:
This is valid if we force the user to have mvn on the PATH, and this will prob cover a vast majority of our cases, but it might be worthwhile allowing users to set, or pass in a mvn_home path